extension/AdcCore.py

```diff
@@ -249,14 +249,14 @@ def agrees_with_os_platform(self):
         if sys.platform == "linux":
             return get_platform() == self.platform
         elif sys.platform == "darwin":
-            os_platform_tag = get_platform()
+            os_platform_tag = distutils.util.get_platform()
             _, os_version, _ = os_platform_tag.split('-')
```

Comment:
Actually since you cannot change the code for the `self.platfrom` I think I would vote for keeping the `get_platform` here as well and making both blocks identical.